### ISSUE TITLE
[Qt5 5.11.3] File was compiled ahead of time with an incompatible version 

### ISSUE THREAD

**Issue by 5schatten** [mmatyas/pegasus-frontend#357]
Since https://github.com/qt/qt5/releases/tag/v5.11.3 & https://github.com/qt/qt5/releases/tag/v5.12.0 have been released can you bump your versions too?

```
[93m[w] QQmlApplicationEngine failed to load component[0m
[93m[w] qrc:/frontend/main.qml:-1 File was compiled ahead of time with an incompatible version of Qt and the original file cannot be found. Please recompile
```

**Comment by mmatyas** [comment]
Yes, I do plan to update the releases to 5.12 in a few days. But how exactly did you produce this error? If you're building Pegasus manually, the QMLs also build with the same Qt version, there are no precompiled binaries in the repository. Perhaps doing a clean build would help?

**Comment by VolcanoMachine** [comment]
i solved this problem simply by clean
